test/parallel/test-dns.js

```diff
@@ -62,6 +62,22 @@ assert(existing.length > 0);
   ]);
 }
 
+{
+  // Various invalidities, all of which should throw a clean error.
+  const invalidServers = [
+    ' ',
+    '\n',
+    '\0',
+    '1'.repeat(3 * 4),
+    ':'.repeat(100000),
+    '['.repeat(100000) // REDOS
+  ];
+  invalidServers.forEach((serv) => {
+    // The error type is undocumented.
+    assert.throws(() => dns.setServers([serv]), /Error.*ERR_INVALID_IP_ADDRESS/, `Unexpected error thrown for ${serv}`);
```

Comment:
This line is too long, but linter complains if I try to break it up?

Reply:
It depends on how you break it up. The following will definitely work.

```js
assert.throws(
  () => foo,
  /abc/,
  'bar'
);
```